File: commands/ping.py
#!/usr/bin/env python3
import discord


async def ping(ctx):

    latency = int(ctx.bot.latency * 1000)

    embed = discord.Embed(color=0xDC2E45, title=f"🏓  Pong! Latency is {latency}ms")

    await ctx.respond(embed=embed)

File: commands/ping.ts
import {
  DiscordInteraction,
  InteractionResponse,
  Colors,
} from "../types/discord";
import { embedResponse } from "../utils/discord";

export async function handlePing(
  _interaction: DiscordInteraction,
): Promise<InteractionResponse> {
  // Serverless doesn't maintain a WebSocket connection, so we can't measure latency
  // like the Python bot could. Just respond with pong.
  return embedResponse({
    title: "🏓 Pong!",
    color: Colors.RED,
  });
}
